great_tables/_gt_data.py

```diff
@@ -177,6 +183,50 @@ def render_formats(self, data_tbl: TblData, formats: list[FormatInfo], context:
 
         return self
 
+    def migrate_unformatted_to_output(
```

Comment:
let's move this out to another file (and make a function), so it isn't a method exposed to users

Reply:
Now moved to `_utils.py`.